src/components/App/App.component.jsx

```diff
@@ -1,56 +1,25 @@
-import React, { useLayoutEffect } from 'react';
+import React, { useState } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
 import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import VideoDetail from '../../pages/VideoDetail/VideoDetail.page';
```

Comment:
Aquí podríamos quitar lo último de VideoDetail.page porque ya tenemos un index.js
```suggestion
import VideoDetail from '../../pages/VideoDetail';
```